src/passes/Inlining.cpp

```diff
@@ -730,6 +747,17 @@ struct FunctionSplitter {
   // staying constant.
   std::unordered_map<Name, Split> splits;
 
+  bool outlinedFunctionWorthInlining(FunctionInfo& origin, Index sizeEstimate) {
+    FunctionInfo info;
+    // This is not accurate, for example the origin function may have
+    // loop or calls even though this section may not have.
+    // However since this is more conservative and it is a practical
```

Comment:
```suggestion
    // This is a conservative estimate, that is, it will return true only when
    // it should, but might return false when a more precise analysis would
    return true. And it is a practical
```